style/amenity-points.mss

```diff
@@ -593,6 +593,13 @@
     marker-clip: false;
   }
 
+  [feature = 'healthcare_vaccination_centre'][zoom >= 15] {
+    marker-file: url('symbols/amenity/hospital.svg');
```

Comment:
The symbol needs to be changed, as discussed, either to a generic icon which can be used for a number of healthcare features which are currently not yet rendered (e.g. laboratories, test centres, birth centres, midwives, nurses, chiropractors, acupuncture, etc) or a unique symbol needs to be created which is recognizable, e.g. a syringe.